src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -196,6 +209,13 @@ def main():
         )
         _create_file_from_dataframe(_get_attendance_events(section_id), file_path)
 
+    if input_directory is not None:
+        system_activities_output_dir = lms.get_system_activities_file_path(schoology_output_path)
+        _create_file_from_dataframe(
+            _get_system_activities(input_directory),
```

Comment:
Once the inner function is eliminated, just pass the function name here without actually calling the function.
```suggestion
            _get_system_activities,
```

Reply:
✔️